HotSpot/View Controllers/AddCarViewController.m

```diff
@@ -0,0 +1,67 @@
+//
+//  AddCarViewController.m
+//  HotSpot
+//
+//  Created by aaronm17 on 7/23/19.
+//  Copyright © 2019 aodemuyi. All rights reserved.
+//
+
+#import "AddCarViewController.h"
+#import "Parse/Parse.h"
+#import "Car.h"
+
+@interface AddCarViewController ()
+
+@property (weak, nonatomic) IBOutlet UIButton *defaultButton;
+@property (weak, nonatomic) IBOutlet UIImageView *carImage;
+@property (weak, nonatomic) IBOutlet UITextField *licensePlate;
+@property (weak, nonatomic) IBOutlet UITextField *carColor;
+
+@end
+
+@implementation AddCarViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view.
+    [self.defaultButton setSelected:NO];
+
+}
+
+- (void)addCar {
+    BOOL *selected = (BOOL *) [self.defaultButton isSelected];
+    Car *car = [Car new];
+    car.licensePlate = self.licensePlate.text;
+    car.carColor = self.carColor.text;
+    [Car addCar:self.carImage withColor:self.carColor.text withLicense:self.licensePlate.text withDefault:selected withCompletion:^(BOOL succeeded, NSError * _Nullable error) {
+        [self.delegate didAddCar:car];
+    }];
+}
+- (IBAction)didTapCancel:(UIBarButtonItem *)sender {
+    [self dismissViewControllerAnimated:YES completion:nil];
+}
+
+- (IBAction)didTapConfirm:(UIBarButtonItem *)sender {
+    [self addCar];
+    
+}
+
+- (IBAction)didTapDefault:(UIButton *)sender {
+    if ([self.defaultButton isSelected]) {
```

Comment:
I think you can simplify this if-else logic to:
self.defaultButton.selected = !self.defaultButton.isSelected

Question: if this is the default car and the user taps on it again, you want to remove the default status on this car? then which car becomes the default? I feel like tapping on default should do nothing if this car is already the default.

Reply:
this is for adding a car. This functionality is just to allow the user to decide whether or not they want it to be the default (tapping on it when it's selected will create it without it being the default car, tapping on it when it's not selected will cause it to be the default car upon creation)